news/2022-09-16-project-loved-september-2022.md

```diff
@@ -0,0 +1,341 @@
+
+While this is an older chart, it was well-known for its LN difficulty back then. However, the community has advanced very far in LN proficiency since 2018, and many more players are capable of playing this than before. We believe this chart should be given a fair shot at voting due to its legacy in that regard.
+
+If you want to see *Ragnarok* in the Loved section, don't forget to support it in voting!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1182067.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Moe Shop - Audiovisual](https://osu.ppy.sh/beatmapsets/1182067#mania) by [Paturages](https://osu.ppy.sh/users/1375479)\
+128 BPM, 1:49 | 3.22★, 3.23★, 3.23★\
+*description written by [-NoName-](https://osu.ppy.sh/users/10400730)*
+
+Our very own Paturages is overdue for a Loved nomination, and one of his SV works, *Audiovisual*, is finally entering the Loved section!
+
+I hope you are familiar with swing rhythms, because there are plenty of those to go around. This chart is heavy on memorization, so if you want a good score on this, be prepared to practice it for a while. If you're not a fan of the SVs, there's still the NSV difficulty for you to simply enjoy the song with.
+
+Support Paturages and vote for *Audiovisual* if you want to see this mapset in the Loved section!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/683291.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Chaos City Niigata - Ukiyoe Yokochou](https://osu.ppy.sh/beatmapsets/683291#mania) by [Imperial Wolf](https://osu.ppy.sh/users/9080296)\
+192 BPM, 2:04 | 4.81★\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+Imperial Wolf is primarily known for his 6K charting, but here we see an impressive jazzy 7K map from him - *Ukiyoe Yokochou*.
+
+Although not very dense, this map is very fast-paced and certainly interesting when it comes to patterning. There are numerous swing rhythms and uneven chords that are intertwined by rolly chordstreams and doublestairs. There are also some minor LNs that, while relatively easy, cannot be ignored and will definitely challenge the player's ability to get good accuracy. The whole map is a quick and intense mid-level experience that may be quite hard to high score on — but very rewarding to those who can accomplish such a feat.
+
+Be sure to check this great chart and cast your vote on *Ukiyoe Yokochou*!
+
+[![](/wiki/shared/news/2022-09-16-project-loved-september-2022/1114344.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Sangatsu no Phantasia - Pastel Rain](https://osu.ppy.sh/beatmapsets/1114344#mania) by [quicalid4](https://osu.ppy.sh/users/6172698)\
+135 BPM, 3:15 | 8.13★\
+*description written by [\_underjoy](https://osu.ppy.sh/users/2235750)*
+
+quicalid4 is known for his smooth and effective charting of 7K LN maps, and *Pastel rain* is one of his more popular and liked works.
```

Comment:
```suggestion
quicalid4 is known for his smooth and effective charting of 7K LN maps, and *Pastel Rain* is one of his more popular and well-regarded works.
```

Reply:
Pastel rain is supposed to be lowercase as per
![image](https://user-images.githubusercontent.com/24534022/190916476-1cb0bbdf-0905-453f-bb65-a165e7321596.png)

Reply:
then it should be fixed for all instances, and the image needs to be updated

Reply:
I think you've read what he said backwards @0x84f , the title is capitalized because metadata person changed it, so that's the one we use in the post. will fix the description

Reply:
oh I didn't think they could change the metadata for the map on the site, thought they just did it on loved.sh

Reply:
that's right. am i missing something?